cloud-chatops/src/read_data.py

```diff
@@ -4,16 +4,16 @@
 import sys
 import os
 import json
-from custom_exceptions import (
+from errors import (
     RepositoriesNotGiven,
     UserMapNotGiven,
     TokensNotGiven,
 )
 
-PATH = "/usr/src/app/cloud_chatops_secrets/"
+PATH = "/usr/src/app/dev_cloud_chatops_secrets/"
```

Comment:
Why is this now explicitly dev? Might be worth a comment explaining how prod vs dev secrets get injected